### DISCUSSION TITLE
Option to disable letterboxing? (feature request?)

### DISCUSSION THREAD

**Discussion by LeoSeyers** [vimeo/player.js#1008] [Feature requests]
Hello,

I was wondering if there is any option available during instanciation to disable the letterboxing / pillar box?
The idea would be to use the video in "cover" mode as you would use it with an image object-fit property for a good fullscreen experience (while preserving the video original ratio).  Is this a feature you would consider implementing in the future?

Alternatively I'm thinking in a way to implement it.
The simplest solution I came up with was to get the video sizing and upscale the iFrame accordingly (+ having an additional portrait or landscape video check) and offsetting its position while hiding overflow on the parent container.

Have a good day
Leo

**Comment by benkingcode** [comment]
Also desperately need this! Just being able to set `object-fit: cover` on the Vimeo video tag
